README.md

```diff
@@ -1,15 +1,15 @@
-# MethHelper
-A mod for Payday2 which notifies the player as to which meth ingredient is to be used.  Requires <a href="http://paydaymods.com/download/">BLT</a>.
+# Meth Helper
```

Comment:
"MethHelper" is the actual name of the tool so it should probably stay the way it is.